LibreNMS/OS/Routeros.php

```diff
@@ -112,19 +112,21 @@ public function discoverWirelessFrequency()
         $sensors = [];
         foreach ($data as $index => $entry) {
             if ($entry['mtxrWlApFreq'] == null) {
-                return $this->discoverSensor(
+                $sensors[] = $this->discoverSensor(
```

Comment:
Ok, these changes are wrong, because `discoverSensor()` loops and return an sensor array itself.  
I fail too see the logic of the code tho.
ping @murrant, you're more clever than me